### ISSUE TITLE
pkg_resources.DistributionNotFound: The 'pycountry' distribution was not found

### ISSUE THREAD

**Issue by martin3000** [pycountry/pycountry#106]
I'm running home assistant and home assistant is using pycountry as an internal dependency. I.e. pycountry is not installed globally with pip.
Inside \_\_init__.py, I get the following exception:

```
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/nuc/.local/lib/python3.8/site-packages/pkg_resources/__init__.py", line 466, in get_distribution
    dist = get_provider(dist)
  File "/home/nuc/.local/lib/python3.8/site-packages/pkg_resources/__init__.py", line 342, in get_provider
    return working_set.find(moduleOrReq) or require(str(moduleOrReq))[0]
  File "/home/nuc/.local/lib/python3.8/site-packages/pkg_resources/__init__.py", line 886, in require
    needed = self.resolve(parse_requirements(requirements))
  File "/home/nuc/.local/lib/python3.8/site-packages/pkg_resources/__init__.py", line 772, in resolve
    raise DistributionNotFound(req, requirers)
pkg_resources.DistributionNotFound: The 'pycountry' distribution was not found and is required by the application
```

This is not catched by the "except" statement.

So instead using 
```
try:
except ImportError:
```
it should be
```
try:
except:
```

**Comment by benelgiac** [comment]
Running into a similar error. #52 should have fixed it, unfortunately it didn't make it in 22.3.5.

Hope the author has plans to tag and release 22.3.6 soon :)

**Comment by jorenvh1** [comment]
@ctheune Any updates on this? Would love to see this in a new release.

**Comment by nschimmoller** [comment]
@martin3000 @jorenvh1 this should resolved with the release of 23.12.7 which includes #52 

Please upgrade using 

`pip install pycountry --upgrade`
